qi-doc/scribblings/qi.scrbl

```diff
@@ -36,5 +36,6 @@ This site hosts @emph{user} documentation. If you are interested in contributing
 @include-section["macros.scrbl"]
 @include-section["field-guide.scrbl"]
 @include-section["principles.scrbl"]
+@include-section["under-the-hood.scrbl"]
```

Comment:
I recommend this come last and be clear that it is more like developer documentation than necessary for an end-user to understand.